dev-embedded/arduino/arduino-1.8.19-r1.ebuild

```
@@ -0,0 +1,121 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 desktop xdg-utils
```

```suggestion
inherit java-pkg-2 desktop xdg
```
I'll do this in a follow-up commit